dev/small-requirements.txt

```diff
@@ -1,26 +1,17 @@
 ## Small test reqs
-# Required
-azure-storage-blob==12.3.0
-# Required to log artifacts and models to GCS artifact locations
-google-cloud-storage==1.14.0
-# Required to log artifacts and models to AWS S3 artifact locations
-botocore==1.12.84  # pinned for moto
-boto3==1.9.84      # pinned for moto
-# Required to log artifacts and models to HDFS artifact locations
-pyarrow==0.17.0
```

Comment:
@apurva-koti looks like we removed these comments without re-introducing them in `setup.py`. Can we make sure that these comments are still captured?